### ISSUE TITLE
[A11y] Explore Collections - Grid - Reflow (Zoom / Mobile view)

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1890]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

Extreme zooming fails, as does mobile view.  Multiple elements' text gets cut off, and in some cases overlaps other text.  There are usability issues as well, because the "masonry" view tries to force three items across instead of stacking more nicely, and the pagination area is similarly "weird".

Text clipping (the "l" in "Centennial" and the "r" in "Barometer"):

![image](https://user-images.githubusercontent.com/2157974/129254482-3b764551-3910-4372-901b-8fbe93c4d0cf.png)

Text overlap and odd search field "squash":

![image](https://user-images.githubusercontent.com/2157974/129254577-3d8b4e0c-79d3-4f28-8608-0095bd65ea83.png)

Pagination:

![image](https://user-images.githubusercontent.com/2157974/129254602-d9247b7d-0c5a-4a3c-8c49-1ae2bfeb543b.png)

### Suggested Fix

This whole view seems very painful for mobile users and could do with a massive redesign.  The table view might be considered an acceptable alternative representation (still haven't audited that, though).

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

View on a phone and on a desktop at 1280x1024, 400% zoom
